bus-mapping/src/eth_types.rs

```diff
@@ -202,9 +253,23 @@ impl<'de> Deserialize<'de> for GethExecStep {
     }
 }
 
+/// Helper type built to deal with the weird `result` field added between `GethExecutionTrace`s in
+/// `debug_traceBlockByHash` and `debug_traceBlockByNumber` Geth JSON-RPC calls.
+#[derive(Clone, Debug, Eq, PartialEq, Deserialize, Serialize)]
```

Comment:
Requiring `Serialize` here is strange; maybe this is an oversight from the ethers-providers crate: to deserialize a json-rpc response it shouldn't require serialize!

Reply:
I've made a PR for this in ethers-rs https://github.com/gakonst/ethers-rs/pull/590

Reply:
We can take this out as soon as a new version is out. Will file an issue to keep that in mind!